app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailFragment.java

```diff
@@ -185,6 +186,25 @@ public void onStart() {
         }
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        User user = organizerDetailViewModel.getUser();
+
+        if (user != null) {
+            String firstName = user.getFirstName();
+            String lastName = user.getLastName();
+            String email = user.getEmail();
+
+            if (firstName != null)
+                ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle(
+                    firstName + " " + lastName);
+            else
+                ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle(email);
+        }
```

Comment:
Where was it previously being set?

Reply:
In the XML file.

Reply:
Why wasn't it working?

Reply:
Whenever the fragment was being resumed, the title was getting changed to `OrganizerDetailActivity` because of the `getTitle()` method of the BaseFragment.

Reply:
The `onResume()` method in the BaseFragment:

```
@Override
public void onResume() {
    super.onResume();
    setTitle(getString(getTitle()));
}
```

Reply:
So why not override `getTitle`?

Reply:
`getTitle()` is already overridden and `OrganizerDetailActivity` is being set from there. Is it possible to set the organizer name from there since it takes int resource value and we cannot change values in strings.xml programatically at runtime?

Reply:
Got it